sdk/cosmos/azure-cosmos/azure/cosmos/_models.py

```diff
@@ -0,0 +1,54 @@
+# The MIT License (MIT)
+# Copyright (c) 2023 Microsoft Corporation
+
+from typing import Any
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+
+class BatchOperationResponse:
```

Comment:
It looks like these models were intended to be public?

Reply:
I was basing myself off of the tables implementation since they have a private file for the models they use

Reply:
Additionally - I'm not sure this model even really needs to be here - as we can just return the `operation_response` (`List[Dict[str, Any]]`) directly right? The only other purpose of this models seems to be to bundle the input with the output - which would only be necessary if output order is not maintained.

Reply:
If we _really_ need a model here (e.g. we anticipate additional data being returned per result in future) then we should name it `BatchOperationResult`